<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SonyToolWindow</class>
 <widget class="QMainWindow" name="SonyToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sony Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="masterActiveCheck">
      <property name="text">
       <string>Master</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="queryIdButton">
          <property name="text">
           <string>Query ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string>Unknown ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="statusSenseButton">
          <property name="text">
           <string>Status sense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Unknown status</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="timeSenseButton">
          <property name="text">
           <string>Time sense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="speedSenseButton">
          <property name="text">
           <string>Speed sense</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PhMediaPanel" name="masterPanel" native="true"/>
    </item>
    <item>
     <widget class="QCheckBox" name="slaveActiveCheck">
      <property name="text">
       <string>Slave</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PhMediaPanel" name="slavePanel" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMaster">
    <property name="title">
     <string>Master</string>
    </property>
    <addaction name="actionMaster_GoTo"/>
    <addaction name="separator"/>
    <addaction name="actionMaster_Use_video_sync"/>
   </widget>
   <widget class="QMenu" name="menuSlave">
    <property name="title">
     <string>Slave</string>
    </property>
    <addaction name="actionSlave_GoTo"/>
    <addaction name="separator"/>
    <addaction name="actionSlave_Use_video_sync"/>
   </widget>
   <addaction name="menuMaster"/>
   <addaction name="menuSlave"/>
  </widget>
  <action name="actionMaster_GoTo">
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSlave_GoTo">
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSlave_Use_video_sync">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use video sync</string>
   </property>
  </action>
  <action name="actionMaster_Use_video_sync">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use video sync</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhMediaPanel</class>
   <extends>QWidget</extends>
   <header>PhCommonUI/PhMediaPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
